README.md

```diff
@@ -40,6 +40,11 @@ The DSN has to be in the following format:
 
 For further explanations have a look at the [Doctrine documentation](http://doctrine-orm.readthedocs.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url).
 
+You can also specify the buffer size with an optional cli-option. Add a suffix (KB, MB or GB) to the value for better readability.
+
```

Comment:
Could we add another explanation here, stating that this is the maximum size of an SQL statement generated and that this can be used to deal with different MySQL `max_allowed_packet` values on the importing side?

I am not sure whether "buffer size" is a term that makes sense to everyone.